apps/docusaurus/docs/nodes/validator-node/operator/delegation-pool-operations.md

```diff
@@ -21,13 +21,22 @@ Before initializing a delegation pool, you need to know the delegation pool addr
 
 - Before you create the delegation pool:
   ```bash
-  aptos account derive-resource-account-address --address <owner_address> --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
+  aptos account derive-resource-account-address --address <owner_account_address> --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
   ```
   - The `<SEED>` is a number chosen by you to create the resource account address to host the delegation pool resource. Once you choose a seed, you should use the same value for all following usages.
-- After you create the delegation pool:
-  ```bash
-  aptos account derive-resource-account-address
-  ```
+
+The command above will generate an output like this:
+```
+{
+"Result": "<delegation_pool_address>"
+}
+
+```
+
+Now you have to provide the <delegation_pool_address> to someone from Aptos Labs so they'll whitelist you.
```

Comment:
Hmm... IIUC, this is only true for `testnet`? i.e., in `mainnet` anyone should be able to create a delegation pool and attract delegated stake. 'cc @michelle-aptos, any thoughts?

Two asks:
1. Perhaps wrap this in an `info` block and make it clear that we're only talking about `testnet`, e.g.,:
```
:::tip Testnet delegation pools
For testnet, you should provide the <delegation_pool_address> to someone from Aptos Labs so you can be added to the allowlist.
:::
```
But, let's see what @michelle-aptos says first.
2. Replace `whitelist` with `allowlist`, as that's more consistent with other docs.

Reply:
yes that's correct! allowlisting is only required for testnet